front-end/src/components/editProduct/EditProduct.js

```diff
@@ -0,0 +1,84 @@
+import { useHistory } from 'react-router-dom';
+import { useEffect, useState } from 'react';
+import styled from 'styled-components';
+import { productService } from '../../services/productServices';
+
+const Container = styled.div`
+    width: 80%;
+    min-height: 100vh;
+    background-color: whitesmoke;
+    margin: 0 auto 0 auto;
+
+    h2{
+        padding: 1rem;
+        text-align: center;
+    }
+    form{
+        display: flex;
+        flex-direction: column;
+        width: 35%;
+        margin: 0 auto 0 auto;
+    }
+
+    form input{
+        margin: 1rem;
+        padding: 0.8rem;
+        outline: none;
+        border: none;
+    }
+
+    form button {
+        padding: 0.8rem;
+        width: 25%;
+        margin: 2rem auto 0 auto;
+        cursor: pointer;
+        border: none;
+        background-color: white;
+        border-radius: 0.5rem;
+
+        :hover{
+            background-color: #BBB;
+            color: white;
+        }
+    }
+`;
+
+const AddProduct = (props) => {
+    const redirect = useHistory();
```

Comment:
Don't think that redirect is the right word here. Better change it to history to be more precise.
```suggestion
    const history= useHistory();
```